src/figuresClasses.ts

```diff
@@ -1,19 +1,90 @@
-export interface Figure {
+enum ShapeTypes {
+  triangle,
+  circle,
+  rectangle,
+}
```

Comment:
```suggestion
enum ShapeTypes {
  triangle = 'triangle,
  circle = 'circle',
  rectangle = 'rectangle',
}
```
String enums should be written like this.
Then you can write, for example:
```
shape = ShapeTypes.triangle
```